authui/src/authflowv2/components/settings-item.css

```diff
@@ -55,14 +72,19 @@
     --settings-item__forward-arrow-color: var(--color-neutral-200);
 
     --settings-item__border-color: var(--color-neutral-400);
+
+    :root.dark {
+      --settings-item-content__text-color: var(--color-neutral-200);
+    }
```

Comment:
This is already inside a `:root.dark` block. Do we really need a nested `:root.dark`?

Reply:
Right it is redundant, removed in latest commit